src/Entity/OgMembership.php

```diff
@@ -335,6 +358,10 @@ public static function baseFieldDefinitions(EntityTypeInterface $entity_type) {
       ->setLabel(t('Group entity type'))
       ->setDescription(t('The entity type of the group.'));
 
+    $fields['entity_bundle'] = BaseFieldDefinition::create('string')
+      ->setLabel(t('Group bundle id'))
```

Comment:
id => ID

Reply:
I also fixed another instance here where `ID` was incorrectly capitalized, and used single quotes instead of double quotes.